src/main/java/mainBackend/MainControllerBack.java

```diff
@@ -0,0 +1,7 @@
+package mainBackend;
+
+public class MainControllerBack {
```

Comment:
Trocar o nome de classe para "Main", pois é nome da classe de início padrão.

Reply:
Ponto resolvido.